cmd/docker-generate/manifest_test.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"testing"
+)
+
+func TestGenerateManifestCmd(t *testing.T) {
+	tests := []struct {
+		expectedOutput    string
+		expectedReference string
+		args              []string
+		expectedErr       bool
+	}{
+		{
+			expectedOutput:    "abc",
```

Comment:
Probably we can have some more meaningful strings defined instead of "abc", "def".